scripts/deployment/send_resend_email.py

```
@@ -0,0 +1,133 @@
+    subject: str,
+    html: str,
+) -> dict[str, object]:
+  request = urllib.request.Request(
+      api_url,
+      data=build_payload(sender, recipients, subject, html),
+      headers={
+          "Authorization": f"Bearer {api_key}",
+          "Content-Type": "application/json",
+      },
+      method="POST",
+  )
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)